### ISSUE TITLE
Uncaught Error: Call to undefined function ot_register_meta_box() in hueman/functions/meta-boxes.php:265

### ISSUE THREAD

**Issue by rabin-io** [presscustomizr/hueman#166]
Upgrading the theme using wp-cli from version 3.0.1 to 3.0.6

```
+--------+-------------+-------------+---------+
| name   | old_version | new_version | status  |
+--------+-------------+-------------+---------+
| hueman | 3.0.1       | 3.0.6       | Updated |
+--------+-------------+-------------+---------+
```
I get this error when trying to access the admin interface, 

```
Notice:  Only variables should be passed by reference in hueman/option-tree/ot-loader.php on line 98

Fatal error:  Uncaught Error: Call to undefined function ot_register_meta_box() in /wp-content/themes/hueman/functions/meta-boxes.php:265
Stack trace:
#0 wp-includes/plugin.php(525): hu_custom_meta_boxes('')
#1 wp-admin/admin.php(168): do_action('admin_init')
#2 wp-admin/index.php(10): require_once('/www/rabin.io/b...')
#3 {main}
  thrown in /wp-content/themes/hueman/functions/meta-boxes.php on line 265

```

**Comment by denzelchia** [comment]
Hi,

Just to add some findings.

With reference to this thread.
https://wordpress.org/support/topic/fatal-erro-after-theme-update?replies=14#post-8268184

I installed the following plugins on my hueman test site.
https://wordpress.org/plugins/advanced-custom-fields/
https://wordpress.org/plugins/types/

Both plugins cause the following fatal error when activated.
Fatal error: Call to undefined function ot_register_meta_box() in /Applications/MAMP/htdocs/hueman-lite/wp-content/themes/hueman/functions/meta-boxes.php on line 265

Hope this information helps in some way.

Thank you

**Comment by Nikeo** [comment]
fixed in v3.0.9
